### ISSUE TITLE
Review-VnVRep-Team32:duplicate test cases

### ISSUE THREAD

**Issue by tangg5** [mehtaj8/Greenway#88]
Regarding map-test-3 and map-test-2, they seem to be testing the same functionality with the same expected output. There are also some redundant test cases that make it overall slightly hard to follow. Also, it might be better to put more information about the background and corresponding requirements to make it easier for people to understand.

**Comment by Utsharga** [comment]
Thank you for the suggestion. The idea behind the two test cases is to test two different features. But further changes will be made to make it more understandable.
